packages/vue/public/assets/storybook/Error/error.svg

```diff
@@ -0,0 +1,16 @@
+<svg width="144" height="144" viewBox="0 0 124 117" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@martaradziszewska why we have 2 `svg` files? Probably move it to shared is will be a good idea, what do you think?